### PR TITLE
Content Manager List view updated

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:

Since the `plugin_content_manager_schema` will change with the upcoming repeatable feature the view needed to be updated a little...

#### My PR is a:

- [X] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-plugin-content-manager/admin/src/components/FilterPickerOption/components.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import styled, { css } from 'styled-components';
+import Bckg from '../../assets/images/background_input.svg';
+
+const Wrapper = styled.div`
+  min-height: 36px;
+  border-left: ${props => (props.borderLeft ? '3px solid #007EFF' : '0px')};
```
Comment (virginieky) [inline packages/strapi-plugin-content-manager/admin/src/components/FilterPickerOption/components.js]: `border-left: ${props => (props.borderLeft && '3px solid #007EFF')};`
could be enough ?

File: packages/strapi-plugin-content-manager/admin/src/containers/ListView/Filter.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Filter.propTypes = {
+  changeParams: PropTypes.func.isRequired,
+  filter: PropTypes.string.isRequired,
+  filters: PropTypes.array.isRequired,
+  index: PropTypes.number.isRequired,
+  isFilterPickerOpen: PropTypes.bool.isRequired,
+  name: PropTypes.string,
+  schema: PropTypes.object.isRequired,
+  toggleFilterPickerState: PropTypes.func.isRequired,
+  value: PropTypes.any,
```
Comment (virginieky) [inline packages/strapi-plugin-content-manager/admin/src/containers/ListView/Filter.js]: Can really be any type or can be oneOfType ?

File: packages/strapi-plugin-content-manager/admin/src/containers/ListView/components.js [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+        &:not(:first-child) {
+          padding: 0;
+        }
+
+        &:first-of-type {
+          margin-top: 2px;
+        }
+
+        &:first-child {
```
Comment (virginieky) [inline packages/strapi-plugin-content-manager/admin/src/containers/ListView/components.js]: add `margin-top: 2px;` here to remove the selector `first-of-type` above

File: packages/strapi-plugin-content-manager/admin/src/containers/ListView/components.js [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+          outline: none;
+        }
+      }
+    }
+  }
+`;
+
+const FooterWrapper = styled.div`
+  margin-top: 2.5rem;
+  // > div {
```
Comment (virginieky) [inline packages/strapi-plugin-content-manager/admin/src/containers/ListView/components.js]: Remove comment if useless

File: packages/strapi-plugin-content-manager/admin/src/containers/SettingViewModel/ListLayout.js [hunk 5]
```diff
@@ -82,6 +82,7 @@ function ListLayout({
 
       moveListField(index, atIndex);
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (virginieky) [inline packages/strapi-plugin-content-manager/admin/src/containers/SettingViewModel/ListLayout.js]: Can be removed if unused ?

File: packages/strapi-plugin-content-manager/admin/src/containers/ListView/Filter.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/containers/ListView/Filter.js]: At some points yes

File: packages/strapi-plugin-content-manager/admin/src/containers/SettingViewModel/ListLayout.js (hunk 5 above)
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/containers/SettingViewModel/ListLayout.js]: No I'd rather keep it for the moment, we need to be really of what deps we put in a hook.

File: packages/strapi-plugin-content-manager/admin/src/components/CustomTable/styledComponents.js [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+`;
+
+const ActionContainer = styled.td`
+  text-align: right;
+
+  i {
+    margin-left: 15px;
+    font-size: 1.1rem;
+    color: #0e1622;
+
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/components/CustomTable/styledComponents.js]: It has one

**Pull request merged**